### ISSUE TITLE
Add Percentage of Changed Reviewers for Each Iteration

### ISSUE THREAD

**Issue by dnozza** [acl-org/arr-health#11]
It would be interesting to see the percentage of new reviewers that replaced the previous ones for each iteration step.

**Comment by SebastinSanty** [comment]
@dnozza I agree, this would be interesting to report. I will add these statistics to the iteration pages. Thank you for the suggestion! :)
